src/actions/metadata.js

```diff
@@ -31,6 +31,14 @@ export function updateFieldKey(namePrefix, fieldKey, newKey) {
   };
 }
 
+export function changeFieldValue(nameAttr, tempValue) {
+  return {
+    type: ActionTypes.CHANGE_FIELD_VALUE,
+    nameAttr,
+    tempValue
+  };
+}
+
```

Comment:
`updateFieldValue` does two things: 
  - update `e.target.value`
  - set `fieldChanged: true` => enable the `Save` button

`changeFieldValue` does two things slightly differently:
  - update `e.target.value`
  - set `fieldChanged: false` => disables an already `enabled` Save` Button. (It will be re-enabled by the above action attached to an `onBlur` event.)

In other words, this new action was added to address our earlier concern regarding dispatching `save` with `<Hotkeys/>`. With this addition, a <TextareaAutosize> under change will deactivate the `Save` button until it goes out of `focus` when `updateValueField` will run to `set` the changed `value` and enable the `Save` button and allow dispatch save action